README.rst

```diff
@@ -51,48 +56,78 @@ optional sleeptime period and produces an output file with the greetings:
        --outputfile outputs/greetings.txt
        --sleeptime 0
    $ cat outputs/greetings.txt
-   Hello John Doe!
    Hello Jane Doe!
+   Hello John Doe!
 
 3. Compute environment
 ======================
 
-In order to be able to rerun the code even several years in the future, we need
-to "encapsulate the current environment", for example to freeze the Python
-version the code is using. We shall achieve this by preparing a `Docker
-<https://www.docker.com/>`_ container image for the program.
+In order to be able to rerun the analysis even several years in the future, we
+need to "encapsulate the current compute environment", for example to freeze the
+Jupyter notebook version and the notebook kernel that our analysis was using. We
+shall achieve this by preparing a `Docker <https://www.docker.com/>`_ container
+image for our analysis steps.
 
 For example:
 
 .. code-block:: console
 
     $ cat environments/python/Dockerfile
-    FROM python:2.7
+    FROM python:3.7
```

Comment:
Using Python 2.7 may actually be nicer here... The goal is to preserve an analysis, and using a asomewhat older Python version can home this message more easily.

Reply:
> The goal is to preserve an analysis 

So true.